.github/workflows/test.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: |
           version=0.0.$GITHUB_RUN_NUMBER
           mvn versions:set -DnewVersion=$version -DgenerateBackupPoms=false
-          echo "::set-output name=version::$version"
+          echo "version=$version" >> $GITHUB_OUTPUT
```

Comment:
Same comment as for the `action-publish-sbt`, it's not clear to me why we even use this in the 1st place.